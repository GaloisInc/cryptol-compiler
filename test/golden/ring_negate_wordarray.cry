main =  negate [0x01, 0x02]
