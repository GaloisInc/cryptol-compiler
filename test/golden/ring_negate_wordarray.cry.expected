[0xff, 0xfe]
